### PR TITLE
Experiments/gmm

### PR DESCRIPTION
**Pull request by nikitautin**

### CODE REVIEW

File: effdet/loss.py [hunk 1]
```diff
@@ -5,13 +5,25 @@
...
 import torch.nn as nn
 import torch.nn.functional as F
 
 from typing import Optional, List, Tuple
 
 
+def _sample_outputs(outputs: torch.Tensor, num_gmm: int) -> torch.Tensor:
+    mean, var, weights = torch.tensor_split(outputs, 3, dim=-1)
+    std = torch.sqrt(torch.sigmoid(var))
+    normal_outputs = std * torch.randn(std.shape, device=std.device) + mean
```
Comment (nick0kcin) [inline effdet/loss.py:19]: Я правильно понимаю, что в папире не было ничего про дополнительный loss, препятствующий тому, чтобы std в 0 уходило?
Просто обычно во всяких баессовских штуках типа VAE есть еще в лоссе прибавка в виде KL дивергенции, которая как раз ограничивает дисперсию, а тут только сигмоида препятствует полному занулению шума

**Review by nick0kcin — COMMENTED** [review]
Вроде просмотрел, нормальных комментов не придумал)

File: effdet/loss.py (hunk 1 above)
Comment (nikitautin) [reply effdet/loss.py:19]: Ага, ничего про это не было написано.

**Review by tdtce — APPROVED** [review]
